internal/core/services/scheduler_manager/scheduler_manager.go

```diff
@@ -140,6 +141,27 @@ func (s *SchedulerManager) CreateNewSchedulerVersionAndEnqueueSwitchVersion(ctx
 	return nil
 }
 
+func (s *SchedulerManager) PatchSchedulerAndSwitchActiveVersionOperation(ctx context.Context, schedulerName string, patchMap map[string]interface{}) (*operation.Operation, error) {
+	scheduler, err := s.schedulerStorage.GetScheduler(ctx, schedulerName)
+	if err != nil {
+		return nil, fmt.Errorf("no scheduler found, can not create new version for inexistent scheduler: %w", err)
```

Comment:
Maybe here would be nice to return errors.NewErrNotFound() instead of a simple error, so in the handler it is easier to check the type of the error